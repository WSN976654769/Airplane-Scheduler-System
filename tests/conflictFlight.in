location import locations.csv
flight add monday 9:00 sydney perth 180
flight add Monday 10:00 sydney perth 180  
flight add monday 9:30 sydney perth 180
flight add Monday 14:01 perth sydney 180  
exit