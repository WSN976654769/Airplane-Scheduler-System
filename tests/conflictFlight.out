User: Imported 51 locations.

User: Successfully added Flight 0.

User: Successfully added Flight 1.

User: Scheduling conflict! This flight clashes with Flight 1 departing from Sydney on Monday 10:00.

User: Scheduling conflict! This flight clashes with Flight 1 arriving at Perth on Monday 14:31.

User: Application closed.
